From 43c9f77323dcf2c95ac74dc1130c8d13dc1e40ef Mon Sep 17 00:00:00 2001
From: Gleb Popov <6yearold@gmail.com>
Date: Mon, 1 Dec 2025 17:32:25 +0300
Subject: [PATCH] cname-test: Attempt to fix the fd leakage

Setting server to NULL prevents proper cleanup at the test end.

Fixes #758
---
 avahi-core/cname-test.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/avahi-core/cname-test.c b/avahi-core/cname-test.c
index d245664..694658f 100644
--- a/avahi-core/cname-test.c
+++ b/avahi-core/cname-test.c
@@ -167,7 +167,6 @@ static void server_callback(AvahiServer *s, AvahiServerState state, AVAHI_GCC_UN
         assert(avahi_test_case_function);
         avahi_test_case_function();
         avahi_log_debug("Server configuration complete.");
-        server = NULL;
     }
 }
 
